privacyidea/lib/user.py

```diff
@@ -592,6 +592,21 @@ def delete(self):
 
         return success
 
+    def user_export_dict(self) -> dict:
```

Comment:
please check if we can use the `info()` property, it should return a `dict` as well.